Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -75,6 +75,21 @@ public static class GoogleAuthConsts
         /// <summary>Installed application localhost redirect URI.</summary>
         public const string LocalhostRedirectUri = "http://localhost";
 
+        /// <summary>IAM access token endpoint for service account.</summary>
+        public const string IamServiceAccountEndpointCommonPrefix = "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/";
+
+        /// <summary>IAM access token endpoint for service account.</summary>
+        public const string IamAccessTokenEndpoint = IamServiceAccountEndpointCommonPrefix + "{0}:generateAccessToken";
+
+        /// <summary>IAM signBlob endpoint for service account.</summary>
+        public const string IamSignEndpoint = IamServiceAccountEndpointCommonPrefix + "{0}:signBlob";
+
+        /// <summary>IAM ID token endpoint for service account.</summary>
+        public const string IamIdTokenEndpoint = IamServiceAccountEndpointCommonPrefix + "{0}:generateIdToken";
+
+        /// <summary>IAM scope.</summary>
```

Comment:
It's unclear to me what this means.

Reply:
I updated the comment. It is the IAM scope the source credential needs (so the access token from the source credential has this scope).